diff -r 90c72002b72e src/scdMainWindow.py
--- a/src/scdMainWindow.py	Tue Oct 23 17:13:12 2007 +0200
+++ b/src/scdMainWindow.py	Tue Oct 30 17:11:34 2007 -0400
@@ -97,9 +97,7 @@ class scdMainWindow:
             if not self.dateBackend.writeDateConfig (sysDate, sysTime):
                 self.showErrorDialog (_("Failed to set time and date."))
                 return
-            if not self.dateBackend.syncHardwareClock ():
-                self.showErrorDialog (_("Failed to synchronize hardware clock."))
-                return
+            self.dateBackend.syncHardwareClock ()
             self.closeParent = True
             self.dateBackend.chkconfigOff ()
 
